packages/wallets/src/features/cfd/flows/Verification/Verification.tsx

```diff
@@ -281,7 +294,7 @@ const Verification: FC<TVerificationProps> = ({ selectedJurisdiction }) => {
                                   })
                                 : () => (
                                       <WalletButton
-                                          disabled={isNextDisabled(context)}
+                                          disabled={isNextDisabled(context) || !isDetailsVerified}
```

Comment:
Have you tried putting the validations inside `isNextDisabled`? The `context` object passed in it, you can get the `formValues` and `errrors` context

Reply:
Updated implementation, please re-review 👍